samples/osx/cocoapods/Podfile

```
@@ -2,7 +2,7 @@ platform :osx, '10.12'
 
 target 'WalletCoreExample' do
   use_frameworks!
-  pod 'TrustWalletCore', '~> 2.0.0'
+  pod 'TrustWalletCore', '~> 2.1.0'
```

```suggestion
  pod 'TrustWalletCore'
```

Can we always use latest?

---

+1